exporters/variables-scss/src/content/generators/fileGenerator.ts

```diff
@@ -32,12 +32,8 @@ const filesData = [
   },
 ];
 
-export const generateFiles = (
-  tokens: Array<Token>,
-  mappedTokens: Map<string, Token>,
-  tokenGroups: Array<TokenGroup>,
-) => {
-  return filesData.map(({ fileName, tokenTypes, groupNames, withCssObject, hasParentPrefix }) => {
+export const generateFiles = (tokens: Array<Token>, mappedTokens: Map<string, Token>, tokenGroups: Array<TokenGroup>) =>
+  filesData.map(({ fileName, tokenTypes, groupNames, withCssObject, hasParentPrefix }) => {
```

Comment:
ESLint: Expected no linebreak before this expression. (implicit-arrow-linebreak)